src/engine/Settings.cs

```diff
@@ -171,6 +175,16 @@ private Settings()
 
     public int CloudSimulationHeight => Constants.CLOUD_Y_EXTENT / CloudResolution;
 
+    /// <summary>
+    /// Property with backing field used to return the default locale for this computer if nothing has been set
+    /// </summary>
+    [JsonIgnore]
+    public string SelectedLanguage
+    {
+        get => selectedLanguage ?? OS.GetLocale();
```

Comment:
I kinda don't like how this property in the settings works.
I'd prefer if `OS.GetLocale()` would get called by the OptionsMenu, or add a new method to Settings to get name of the selected language (even if they are null) if it will be useful to get the current language as string.